src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -499,6 +500,7 @@ private async Task PerformAuditAsync(IEnumerable<RestoreTargetGraph> graphs, Tel
 
             telemetry.TelemetryEvent[AuditLevel] = (int)audit.MinSeverity;
             telemetry.TelemetryEvent[AuditMode] = AuditUtility.GetString(audit.AuditMode);
+            if (audit.SuppressedAdvisories.Count() > 0) { telemetry.TelemetryEvent[SuppressedAdvisories] = ConcatAsString(audit.SuppressedAdvisories); }
```

Comment:
```suggestion
            telemetry.TelemetryEvent[SuppressedAdvisories] = ConcatAsString(audit.SuppressedAdvisories);
```

There's value in knowing when the count is zero vs telemetry from an older version of VS before this property/feature exists.

Also, never use LINQ when you don't need to. HashSet has a Count property, so `.Count > 0` is going to cause fewer allocations, therefore fewer garbage collections, and just generally have less CPU work to do than `Count()`.

Reply:
I've fixed this now, thanks!